src/test/projects-tests/one-module-with-parent/pom.xml

```diff
@@ -51,4 +51,19 @@
       <url>file://${user.home}/mavenreposnapshot</url>    
     </snapshotRepository>
   </distributionManagement>  
+  
+  <!-- Required to discover Parent POM if it is not cached -->
```

Comment:
In fact it is cached in your local repository but Maven 3.x doesn't use it because it comes from a remote repository you don't have access to when running tests
https://cwiki.apache.org/confluence/display/MAVEN/Maven+3.x+Compatibility+Notes#Maven3.xCompatibilityNotes-ResolutionfromLocalRepository
https://issues.apache.org/jira/browse/MNG-5181